src/components/AccessControlNew.tsx

```diff
@@ -185,6 +224,7 @@ const AccessControlNew = () => {
                     visible={setupNewRuleVisible}
                     bodyStyle={{paddingBottom: 80}}
                     onClose={onCancel}
+                    autoFocus={true}
```

Comment:
Codacy has a fix for the issue: [Replace `····················` with `··········`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
          autoFocus={true}
```